src/debug.c

```diff
@@ -719,6 +721,7 @@ NULL
         } else if (!strcasecmp(name,"double")) {
             addReplyDouble(c,3.14159265359);
         } else if (!strcasecmp(name,"bignum")) {
+            /* Not currently supported. */
```

Comment:
maybe a better comment would be "not currently in use".
by "supported" you meant redis-cli, and we don't necessarily care.
anyway, i don't think we care to document that, one day we'll add support, till then, this debug tool is the only way to generate that type.
```suggestion
```